linux_os/guide/system/accounts/accounts-physical/require_singleuser_auth/tests/rhel7_correct_value.pass.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# platform = Red Hat Enterprise Linux 7
+
+{{% if init_system == "systemd" -%}}
+service_file="/usr/lib/systemd/system/rescue.service"
+sulogin='/bin/sh -c "/usr/sbin/sulogin; /usr/bin/systemctl --fail --no-block default"'
+if grep "^ExecStart=.*" "$service_file" ; then
+    sed -i "s%^ExecStart=.*%ExecStart=-$sulogin%" "$service_file"
+else
+    echo "ExecStart=-$sulogin" >> "$service_file"
+fi
+{{%- else -%}}
```

Comment:
This is test scenario for RHEL7 so pelase remove the whole part in the `else`, it is not needed.

Reply:
done